cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -2132,6 +2263,31 @@ def drop_data(self, ids: SequenceNotStr[NodeId]) -> int:
         # Nodes will be deleted in .delete call.
         return 0
 
+    @classmethod
+    @lru_cache(maxsize=1)
+    def get_write_cls_parameter_spec(cls) -> ParameterSpecSet:
```

Comment:
All these `classmethod`s do not seem to use the class.

Reply:
The parent implementation uses the class.